docs/manual/manual_buzzer_control.rst

```diff
@@ -0,0 +1,183 @@
+An example of a full calibration song is played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/hobby_calibration.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+Timeout
+----------
+When a :ref:`timeout <manual_timeout>` is reached, your module will play the timeout song as specified by its :ref:`timeout_behavior`. 
+When the Song Playback is set to play either once or continuously, you will here the timeout song played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/timeout.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+If the song playback option is set to Continuously, the timeout song will repeat until the timeout is ended.
+
+Watchdog
+-----------
+Watchdog timers are used to indicate that there has been a blocking error in the firmware. When a watchdog error occurs, the module restarts, and plays the watchdog error song played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/watchdog.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+If your module plays the watchdog error song, please reach out to our support team at support@vertiq.co with the details surrounding the watchdog. 
+Any flight logs, support files, etc. that can be provided will help our team best diagnose and fix the issue in a new firmware release.
+
+Buzzer Song Volume
+====================
+If the audio feedback from your module is either too loud or quiet, you can adjust its volume with the Volume Max parameter available in the Control Center's advanced tab.
+
+.. image:: ../_static/manual_images/buzzer_song_mp3s_pics/volume_max.png
+
+To increase the volume, increase the maximum voltage, to decrease the volume, decrease maximum voltage.
+
+Custom Songs with the Buzzer Control Client
+===================================================
+Since the buzzer controller is a :ref:`standard IQUART client <buzzer_control>`, you can communicate with it via IQUART. The examples presented here will be performed using the :ref:`Python API <getting_started_python_api>`.
+
+In order to play a note of your choosing, you must set, in order, a frequency, duration, and volume. Only then can you trigger playback.
+
+Suppose you want your module to play a middle C (261.6 Hz) for 3 seconds. 
+
+.. code-block:: python
+
+    import iqmotion as iq
+    import time
+
+    # Module Communication - This must be updated with the correct parameters for your module's serial communication
+    com = iq.SerialCommunicator("COM3", baudrate=115200)
+
+    # Using the Vertiq4006 with additional custom client files
```

Comment:
Comment says 4006, but the module is a 2306. Also there are no custom client files